okteto.yml

```diff
@@ -1,12 +1,12 @@
 build:
-  todo-list:
-    image: okteto.dev/todo-list
-  
+  go-todo-list:
+    image: okteto.dev/go-todo-list:001
```

Comment:
Since we are using `kustomize`, we should remove the `image` field, and use `kustomize edit set image ${OKTETO_BUILD_TODO_LIST_IMAGE}` instead